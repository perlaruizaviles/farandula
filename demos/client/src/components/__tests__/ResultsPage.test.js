import React from 'react';
import {shallow} from 'enzyme';
import TravelResults from '../Avail/ResultsPage';

describe('Rendering TravelResults ', () => {

  const tree = shallow(<TravelResults />);
  
  it('Should create an snapshot for TravelResults', () => {
    expect(tree).toMatchSnapshot();
  });
  
  it('Renders Correct TravelResults', () => {
    expect(tree.find('Container').length).toBe(1);
  });
});