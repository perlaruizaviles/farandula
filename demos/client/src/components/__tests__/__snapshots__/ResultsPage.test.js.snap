exports[`Rendering TravelResults  Should create an snapshot for TravelResults 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Container>
    <Grid>
        <GridColumn
            floated="left"
            width={3}>
            <Connect(FiltersCard) />
        </GridColumn>
        <GridColumn
            floated="right"
            width={13}>
            <GridRow>
                <Connect(TravelSearch) />
            </GridRow>
            <GridRow>
                <Connect(ItineraryList) />
            </GridRow>
        </GridColumn>
    </Grid>
</Container>,
  "nodes": Array [
    <Container>
      <Grid>
            <GridColumn
                  floated="left"
                  width={3}>
                  <Connect(FiltersCard) />
            </GridColumn>
            <GridColumn
                  floated="right"
                  width={13}>
                  <GridRow>
                        <Connect(TravelSearch) />
                  </GridRow>
                  <GridRow>
                        <Connect(ItineraryList) />
                  </GridRow>
            </GridColumn>
      </Grid>
</Container>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <TravelResults />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Container>
          <Grid>
                    <GridColumn
                              floated="left"
                              width={3}>
                              <Connect(FiltersCard) />
                    </GridColumn>
                    <GridColumn
                              floated="right"
                              width={13}>
                              <GridRow>
                                        <Connect(TravelSearch) />
                              </GridRow>
                              <GridRow>
                                        <Connect(ItineraryList) />
                              </GridRow>
                    </GridColumn>
          </Grid>
</Container>,
        "_debugID": 2,
        "_renderedOutput": <Container>
          <Grid>
                    <GridColumn
                              floated="left"
                              width={3}>
                              <Connect(FiltersCard) />
                    </GridColumn>
                    <GridColumn
                              floated="right"
                              width={13}>
                              <GridRow>
                                        <Connect(TravelSearch) />
                              </GridRow>
                              <GridRow>
                                        <Connect(ItineraryList) />
                              </GridRow>
                    </GridColumn>
          </Grid>
</Container>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <TravelResults />,
}
`;
